src/client/zweb/deployer_old/index.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 const fs = require('fs');
 const logger = require('../../../core/log');
-const log = logger.child({module: 'Deployer'});
+const log = logger.child({module: 'Deployer_old'});
```

Comment:
What do you mean by "majority of cases"? The only place where this deployer is used is in a manual `deploy` command. My deployer is not a blog deployer, now it can be used only for upgradeable contract deployment, but the idea is that it will replace the old one over time.
I can do the opposite and rename the new deployer to `deployer_new` if it makes sense